src/App.svelte

```diff
@@ -28,8 +28,9 @@
 
   const activeRouteStore = router.activeRouteStore;
 
-  void router.loadFromLocation();
-  httpd.initialize();
+  // This allows us to get the correct values for the httpd state and port, before navigation.
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  httpd.initialize().finally(() => void router.loadFromLocation());
```

Comment:
The prevalent convention in the code base seems to be using `void` to disable the no-floating-promises warning:
```suggestion
  void httpd.initialize().finally(() => router.loadFromLocation())
```